CHANGES.md

```diff
@@ -19,6 +19,15 @@
   ...
   ```
 
+### Fixes
+
+* Added `shutdown_on_close=True` parameter to coiled params to ensure that the 
+  clusters are shut down on close. (#881)
+* Introduced new parameter `region` for utility function `new_cluster` in 
```

Comment:
I think this entry should rather not be listed under fixes, as it was not a problem but we want to ensure that per default our prefered region is used.

Reply:
Hmm, yeah, I agree.  I have listed them under other changes. Fine?

Reply:
Yes, fine with this :)